backends/conrod_wgpu/src/lib.rs

```diff
@@ -784,6 +792,20 @@ fn texture_format_to_component_type(format: wgpu::TextureFormat) -> wgpu::Textur
         | wgpu::TextureFormat::Rgb10a2Unorm
         | wgpu::TextureFormat::Depth32Float
         | wgpu::TextureFormat::Depth24Plus
-        | wgpu::TextureFormat::Depth24PlusStencil8 => wgpu::TextureComponentType::Float,
+        | wgpu::TextureFormat::Depth24PlusStencil8
```

Comment:
Do you think it makes sense to handle the TODO comment from above instead of adding to this?

```
// TODO: Remove this when this lands https://github.com/gfx-rs/wgpu/pull/628
```

Reply:
Ah, yes. I didn't notice it, sorry.